### PR TITLE
Documentation for updating vSphere credentials

### PR DESCRIPTION
**Pull request by 2ez4szliu**
*Issue #, if available: #6468

*Description of changes:*

*Testing (if applicable):*

*Documentation added/planned (if applicable):*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: docs/content/en/docs/clustermgmt/cluster-upgrades/vsphere-credential-update.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+    - If annotation `kubectl.kubernetes.io/last-applied-configuration` exists on any of the above Secret object, update password in `kubectl.kubernetes.io/last-applied-configuration` field.
+    -  `{CLUSTER_NAME}-csi-vsphere-config` under `eksa-system` namespace - If annotation `kubectl.kubernetes.io/last-applied-configuration` exists on the secret object, update password in `kubectl.kubernetes.io/last-applied-configuration` field.
+
+2. Only update Secret `vsphere-credentials` under `eksa-system` namespace then trigger a full EKS-A CAPI cluster upgrade by modifying the cluster spec:
+- Update `EKSA_VSPHERE_PASSWORD` environment variable to the new password and get the base64 encoded string of the password using `echo -n "<YOUR_PASSWORD>" | base64`
+- Update secret `vsphere-credentials` under `eksa-system` namespace - Update `password`, `passwordCP`, `passwordCSI` field under data and in `kubectl.kubernetes.io/last-applied-configuration` if annotation exists.
+- Modify cluster spec file and run `eksctl anywhere upgrade cluster -f <cluster-config-file>` to trigger a full cluster upgrade. This will automatically apply the new credentials to all related secrets.
```
Comment (panktishah26) [inline docs/content/en/docs/clustermgmt/cluster-upgrades/vsphere-credential-update.md:19]: Can we specify a few fields a customer can upgrade to enable full CAPI cluster upgrade as an example?

**Review by panktishah26 — COMMENTED** [review]
Thank you for documenting these steps. Your PR looks good. I have one small suggestion.

File: docs/content/en/docs/clustermgmt/cluster-upgrades/vsphere-credential-update.md (hunk 1 above)
Comment (2ez4szliu) [reply docs/content/en/docs/clustermgmt/cluster-upgrades/vsphere-credential-update.md:19]: Thanks Pankti, added some examples

Comment (2ez4szliu) [reply docs/content/en/docs/clustermgmt/cluster-upgrades/vsphere-credential-update.md]: Specified that we can update any field in cluster config to trigger the full cluster upgrade.

**Pull request merged**